### PR TITLE
use table instead of smart table

### PR DESCRIPTION
**Pull request by 0xaptosj**
"Table should be more performant than smart table"
-- Max

### CODE REVIEW

File: move/sources/canvas/token.move [hunk 1]
```diff
@@ -585,16 +585,15 @@ module addr::canvas_token {
...
+            last_contribution_s: table::new(),
             allowlisted_artists,
             blocklisted_artists,
             admins,
             created_at_s,
             extend_ref,
             mutator_ref,
         };
-        move_to(&object_signer, new_canvas_);
-        smart_table::destroy(last_contribution_s);
```
Comment (0xaptosj) [inline move/sources/canvas/token.move:597]: table doesn't expose `destroy` as its destroy is friend so it's not callabled 
```
public(friend) fun destroy<K: copy + drop, V>
```

**Comment by banool** [comment]
In addition to the problems you're talking to above, this might break the indexer processor, perhaps let's not land this until we know we need to.

File: move/sources/canvas/token.move [hunk 2]
```diff
@@ -725,7 +724,7 @@ module addr::canvas_token {
         let object_signer = object::generate_signer_for_extending(&extend_ref);
         let new_canvas_ = Canvas {
             config,
-            pixels: smart_table::new(),
+            pixels: table::new(),
```
Comment (CapCap) [inline move/sources/canvas/token.move:727]: this might be better as smart table: please try to do some benchmarking to determine :-)

Comment (CapCap) [reply move/sources/canvas/token.move:727]: cc @banool for your thoughts, and if you already did the research :-)

Comment (banool) [reply move/sources/canvas/token.move:727]: I did actually, quite extensively, check it out: https://www.notion.so/aptoslabs/Experimenting-with-different-vectors-map-like-structures-a380147e5f564411b4a199ef58b9f65e 🤠

Comment (CapCap) [reply move/sources/canvas/token.move:727]: Amazing :-) thank you, I think this is a perfect framework for @0xaptosj to use for table

**Pull request closed without merging**